src/main/java/org/folio/circulation/domain/notice/NoticeFormat.java

```diff
@@ -5,6 +5,8 @@
 public enum NoticeFormat {
 
   EMAIL("Email", "email", "text/html"),
+  PRINT("Print", "print", "text/html"),
```

Comment:
My understanding is that this change is for batching email notices together. Why are these additional formats needed for this?

Reply:
This is correct. I added them just to match circulation-storage schema, where `Print` and `SMS` are allowed values. Would you like to remove these changes as they are not necessary now?

Reply:
If they are going to be treated the same as unknown, I don't really understand the value in having them.

If it makes it easier to understand the mapping between the two contexts (storage and business logic) then I think they can remain.

I'm trying to ask questions about why changes are made to understand the decisions better and to encourage reflection, for something like this, I leave the decision to you.

Reply:
Given these formats are not supported now, it seems like there is no point in them.  ISo I'm going to remove them for now.